modules/client-common/src/main/java/org/apache/ignite/client/proto/query/event/BatchExecuteResult.java

```diff
@@ -30,12 +30,30 @@
     /** Update counts. */
     private int[] updateCnts;
 
+    /** Error code. */
+    private int errorCode;
+
     /**
      * Constructor.
      */
     public BatchExecuteResult() {
     }
 
+    /**
+     * Constructor.
+     *
+     * @param status Status code.
+     * @param errorCode Product error code.
+     * @param errorMessage Error message.
+     * @param updateCnts Array with success and error markers.
+     */
+    public BatchExecuteResult(int status, int errorCode, String errorMessage, int[] updateCnts) {
```

Comment:
The constructor looks ambiguous. 
Is it possible to get a error and result at the same time?